starter-code/public/index.html

```diff
@@ -38,8 +37,8 @@
         <div class="icon-menu"></div>
         <ul>
           <!-- TODO: Refactor the href attributes in the Home and About anchor tags to navigate users to our new client-side routes -->
-          <li class="tab" data-content="articles"><a href="#" class="icon-home"> Home</a></li>
-          <li class="tab" data-content="about"><a href="#" class="icon-address-book"> About</a></li>
+          <li class="tab" data-content="articles"><a href="./home" class="icon-home"> Home</a></li>
+          <li class="tab" data-content="about"><a href="./about" class="icon-address-book"> About</a></li>
```

Comment:
Note that you'll what to just have a '/'. This is because think about how URLs look, the '.' is usually reserved for '.com','.org', etc. This href is going to be added to your URL.